src/components/Navbar.js

```diff
@@ -22,17 +22,28 @@ const Navbar = () => {
 
   ];
   return (
-    <nav>
-      <ul>
+    <nav className="d-flex flex-wrap justify-content-between align-items-center mx-4 mt-2">
```

Comment:
- [ ] Wrapping the Navbar with a `container` class would probably be better looking. Top margin would also be nice.

For example, **we could do this**:
```
<nav className="container mt-4">
  <div className="d-flex flex-wrap...">
    ...
  </div>
</nav>
```

**To get this**

![navbar](https://user-images.githubusercontent.com/36542986/136092136-ef687bad-3d29-48db-a1db-946a9b3bc746.jpg)